docs/introduction.rst

```diff
@@ -1,77 +1,73 @@
 .. _introduction:
 
-Why ScanCode.io
-===============
+ScanCode.io Overview
+====================
 
-Modern software is built from many open source packages assembled with new code.
-Knowing which free and open source code package is in use matters because:
+**ScanCode.io** is a server to script and automate the process of
+**Software Composition Analysis (SCA)** to identify any open source components
+and their license compliance data in an application’s codebase. ScanCode.io can be
+used for various use cases, such as Docker container and VM composition analyses, among other applications.
 
-- knowing the license of third-party code is required before using it, and
-- you want to avoid using buggy, outdated or vulnerable components.
+Why ScanCode.io?
+----------------
 
-Because it is so easy to include and reuse new code downloaded from the internet,
-it is often surprisingly hard to get a proper inventory of all the third-party
-code origins and licenses used in a software project.
-There are some great tools available to scan your code and help uncover these.
+Modern software is built from many open source packages assembled with new code.
+Knowing which free and open source code package is in use matters because:
 
-And when you reuse only a few FOSS components in a single project, running one
-of these tools (such as the scancode-toolkit) by hand together
-with a spreadsheet may be enough to manage your software composition analysis.
+- You're required to know the license of third-party code before using it, **and**
+- You want to avoid using buggy, outdated or vulnerable components.
 
-But when you scale up, running automated and reproducible analysis pipelines
-that are adapted to a software project unique context and technology platform is
+It's usually convenient to include and reuse new code downloaded from the internet; however, it's often surprisingly hard to get a proper inventory of all third-party code origins and licenses used in a software project.
+There are some great tools available to scan your code and help uncover these details. For example, when you reuse only a few FOSS components in a single project, running one of these tools, such as the **ScanCode-toolkit**, manually along
```

Comment:
It would make sense to have line break at 100 characters, this also fails the tests as the linter chacks for those.

Reply:
Running `make check` locally before pushing would be a good thing to do.